### PR TITLE
AC Test Fix - LRAC-9396 | master

### PR DESCRIPTION
**Pull request by 91nilo**
https://issues.liferay.com/browse/LRAC-9396
Run: https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1564018824

### CODE REVIEW

**Comment by 91nilo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 1]
```diff
@@ -261,7 +261,11 @@ definition {
 
 		ACNavigation.goToSettings();
 
-		ACUtils.assertItemsName(itemList = "${dataSource},Beryl Commerce");
+		ACUtils.assertItemsName(itemList = "${dataSource}");
+
+		AssertElementPresent(
+			locator1 = "ACSettings#PROPERTIES_ASSET_TITLE_LIST",
+			propertyName = "Data Sources");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:268]: Hi @91nilo, I think you can use this macro to check the title

`ACUtils.assertPageTitle(pageTitle = "Data Sources");`

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by christianstokes** [comment]
ci:forward

**Pull request closed without merging**